google/cloud/spanner/CMakeLists.txt

```diff
@@ -234,6 +234,54 @@ add_library(google-cloud-cpp::spanner ALIAS google_cloud_cpp_spanner)
 include(CreateBazelConfig)
 create_bazel_config(google_cloud_cpp_spanner YEAR "2019")
 
+if (GOOGLE_CLOUD_CPP_ENABLE_HTTP_TRANSCODING)
+    add_library(
+        google_cloud_cpp_spanner_rest # cmake-format: sort
+        admin/database_admin_connection.h
```

Comment:
The `admin/database_admin_connection.h` and `admin/instance_admin_connection.h` headers also appear in the gRPC-based library.  That seems problematic: "I included a header, but the functions are in another library..."?

Maybe split the factory function?